src/components/Button.tsx

```diff
@@ -19,9 +20,9 @@ const buttonStyle = (
 ) => css`
   background-color: ${selected ? selectedColor : baseColor};
   color: ${textColor};
-  padding: 6px 12px;
-  font-weight: 600;
-  font-size: 14px;
+  padding: ${theme.spacing.spacing2}${theme.spacing.spacing3};
```

Comment:
```suggestion
  padding: ${theme.spacing.spacing2} ${theme.spacing.spacing3};
```
요렇게 안 들어가도 스타일 잘 먹히나요..?